x-pack/test/functional/page_objects/endpoint_page.ts

```diff
@@ -58,5 +59,13 @@ export function EndpointPageProvider({ getService }: FtrProviderContext) {
             )
         );
     },
```

Comment:
I used the Cheerio service to pull data from a table in my pagination test. I had to use this in conjunction with dataTestSubjects. 👍